### PR TITLE
[MU4] Transpose dialog

### PR DESCRIPTION
**Pull request by RomanPudashkin**
transpose

### CODE REVIEW

File: mu4/notation/view/widgets/transposedialog.h [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+
+#ifndef MU_NOTATION_TRANSPOSEDIALOG_H
+#define MU_NOTATION_TRANSPOSEDIALOG_H
+
+#include "ui_transposedialog.h"
+#include "libmscore/mscore.h"
```
Comment (Eism) [inline mu4/notation/view/widgets/transposedialog.h:24]: Please get rid of this dependency in this file.

File: mu4/notation/view/widgets/transposedialog.h [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+private:
+    void hideEvent(QHideEvent*) override;
+
+    INotationPtr notation() const;
+    INotationSelectionPtr selection() const;
+    INotationInteractionPtr interaction() const;
+
+    Key firstPitchedStaffKey() const;
+
+    void setEnableTransposeKeys(bool val) { transposeKeys->setEnabled(val); }
```
Comment (Eism) [inline mu4/notation/view/widgets/transposedialog.h:62]: Please move the implementation of methods to cpp.

Comment (RomanPudashkin) [reply mu4/notation/view/widgets/transposedialog.h]: fixed

File: mu4/notation/view/widgets/transposedialog.h (hunk 1 above)
Comment (RomanPudashkin) [reply mu4/notation/view/widgets/transposedialog.h]: fixed

**Review by Eism — APPROVED** [review]

File: mu4/appshell/qml/DevTools/NotationDialogs/MU3Dialogs.qml [hunk 3]
```diff
@@ -5,13 +5,30 @@ import MuseScore.Ui 1.0
...
 
-        text: qsTrc("devtools", "Measure properties")
+        spacing: 12
 
-        onClicked: {
-            api.launcher.open("musescore://notation/measureproperties?index=0")
+        FlatButton {
+            width: 140
+
+            text: qsTrc("devtools", "Measure properties")
```
Comment (igorkorsukov) [inline mu4/appshell/qml/DevTools/NotationDialogs/MU3Dialogs.qml:17]: I think, no need to make translatable strings for devtools

**Review by igorkorsukov — APPROVED** [review]

File: mu4/appshell/qml/DevTools/NotationDialogs/MU3Dialogs.qml (hunk 3 above)
Comment (RomanPudashkin) [reply mu4/appshell/qml/DevTools/NotationDialogs/MU3Dialogs.qml]: fixed

**Pull request merged**